src/Tickets/Commerce/Order_Modifiers/Modifier_Admin_Handler.php

```
@@ -199,6 +199,19 @@ public function is_on_page(): bool {
 		return ! empty( $admin_page ) && static::$slug === $admin_page;
 	}
 
+	/**
+	 * Defines whether the current page is the Event Tickets Order Modifiers edit page.
+	 *
+	 * @since TBD
+	 *
+	 * @return bool True if on the Order Modifiers edit page, false otherwise.
+	 */
+	protected function is_on_edit_page(): bool {
+		$is_edit = tribe_is_truthy( tec_get_request_var( 'edit', '0' ) );
+
+		return $this->is_on_page() && $is_edit;
```

put `$is_edit` first for faster fail fast :D